Services/Diff.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace FroshViewSnapshots\Services;
+
+use FineDiff;
+
+class Diff
+{
+    public function __construct()
+    {
+        require_once implode(DIRECTORY_SEPARATOR, [__DIR__, '..', 'lib', 'finediff', 'finediff.php']);
+    }
+
+    public function diffPlain(string $from, string $to): FineDiff
```

Comment:
Would you mind removing the return type declarations as well as other PHP7 specific type hints? It's just that other components of this plugin don't make use of them to still provide support for PHP5.x. Plus I think it's a little inconsistent to have them but nowhere else. In the future I plan to drop support for PHP5.6 all together, just not right now.

Reply:
I expected to be able to use them as the README.md stated that this is a PHP7 project. If it is not I can remove it and therefore add PHPDocs.